### PR TITLE
restyle README.md

### PR DESCRIPTION
**Pull request by yiwenwang2090**
Hi, I add a new commit to restyle README.md
I added some titles and emphasis to make the instructions easier to follow.

### CODE REVIEW

**Comment by rduivenvoorde** [comment]
Hi, thanks for taking time with this!

Are the ```<h3>``` tags actually markdown? Or are we mixing up html and markdown in this case?
Because in the latter case, I would prefer to use `#` till `####` etc ?

Another question: these instructions are pretty old, did you try some of them? Please let us know if (parts/some) of them just don't work anymore.

Thanks again.

File: README.md [hunk 1]
```diff
@@ -23,17 +23,19 @@ Note: you will use a QGIS docker image from hub.docker.com, created with the doc
 
 https://github.com/qgis/QGIS-Sysadmin/blob/master/docker/sphinx/Dockerfile-html
 
-First: install Docker.
+<h3>1.Install Docker</h3>
```
Comment (agiudiceandrea) [inline README.md]:
```suggestion
<h3>1. Install Docker</h3>
```

File: README.md [hunk 2]
```diff
@@ -64,36 +67,39 @@ either your own fork or the original QGIS repo like here:
...
+<h4>Actual command lines:</h4>
 
-On linux (use your own repo path here!):
+On ***linux***, use your own repo path here!:
 
     # english html
     docker run -t -i -v /home/richard/dev/QGIS-Website:/QGIS-Website -w=/QGIS-Website --rm=true qgis/sphinx_html_3 make html
 
-On windows (tested on Win7 and Win8), use your own repo path here!
+On ***windows*** (tested on Win7 and Win8), use your own repo path here!
```
Comment (agiudiceandrea) [inline README.md]:
```suggestion
On ***Windows*** (tested on Win7 and Win8), use your own repo path here!
```

File: README.md [hunk 3]
```diff
@@ -111,8 +119,8 @@ Besides this, you can also have a look into the scripts docker-run.sh and docker
 
 Building the website using Make
 -------------------------------
-
-Building is only tested on Linux systems using make, on windows we now started a Paver setup (see below)
+<h3>1. Setup</h3>
+Building is only tested on Linux systems using make, on ***windows*** we now started a Paver setup (see below)
```
Comment (agiudiceandrea) [inline README.md:123]:
```suggestion
Building is only tested on Linux systems using make, on ***Windows*** we now started a Paver setup (see below)
```

File: README.md [hunk 4]
```diff
@@ -64,36 +67,39 @@ either your own fork or the original QGIS repo like here:
...
+```-w=/QGIS-Website``` means that it is to be used as the working dir of Docker
 
-"--rm=true" means remove the container after the build
+```--rm=true``` means remove the container after the build
 
-Now the actual command lines:
+<h4>Actual command lines:</h4>
 
-On linux (use your own repo path here!):
+On ***linux***, use your own repo path here!:
```
Comment (agiudiceandrea) [inline README.md]:
```suggestion
On ***Linux***, use your own repo path here!:
```

File: README.md [hunk 5]
```diff
@@ -102,7 +108,9 @@ The contents of this file should be like this:
     token = 
     username = yourusernamehere
 
-Copy this file to the root of your repo. With me, that is /home/richard/dev/QGIS-Website. NOW you can run it using 'make full' and a LANG parameter like this:
+Copy this file to the root of your repo. With me, that is /home/richard/dev/QGIS-Website. 
+<h3>6. Run</h3>
+NOW you can run it using 'make full' and a LANG parameter like this:
 
     # french html (linux)
```
Comment (agiudiceandrea) [inline README.md]:
```suggestion
    # french html (Linux)
```

File: README.md [hunk 6]
```diff
@@ -214,12 +226,12 @@ General use:
 To be able to build localized versions of the Website with paver the
 'Transifex-client (tx)' is needed.
 
-On linux, install with::
+On ***linux***, install with::
```
Comment (agiudiceandrea) [inline README.md]:
```suggestion
On ***Linux***, install with::
```

**Review by agiudiceandrea — CHANGES_REQUESTED** [review]

**Pull request closed without merging**